packages/web-app-files/src/components/BatchActions.vue

```diff
@@ -227,11 +259,24 @@ export default {
 
     isEmpty() {
       return this.activeFilesCurrentPage.length < 1
+    },
+
+    isDangerVariation() {
```

Comment:
The `is` in the computed prop name is confusing (same for the next two props), because you don't return a boolean but a string.

Reply:
Good hint, thanks